redaxo/src/addons/media_manager/lib/effects/effect_img2img.php

```diff
@@ -0,0 +1,116 @@
+<?php
+
+/**
+ * @package redaxo\media-manager
+ */
+
+class rex_effect_img2img extends rex_effect_abstract
+{
+    private static $convert_types = [
+        'jpg',
+        'png',
+        'gif',
+        'webp',
+    ];
+
+    private static $convert_to = [
+        'jpg' => [
+            'ext' => 'jpg',
+            'content-type' => 'image/jpeg',
+        ],
+        'png' => [
+            'ext' => 'png',
+            'content-type' => 'image/png',
+        ],
+        'gif' => [
+            'ext' => 'gif',
+            'content-type' => 'image/gif',
+        ],
+        'webp' => [
+            'ext' => 'webp',
+            'content-type' => 'image/webp',
+        ],
+    ];
+
+    private static $convert_tos = ['jpg', 'png', 'gif', 'webp'];
+    private static $convert_to_default = 'webp';
+
+    public function execute()
+    {
+        $media = $this->media;
+
+        $ext = $media->getFormat();
+        $ext = 'jpeg' === $ext ? 'jpg' : $ext;
```

Comment:
@ynamite Hier sollte "jpeg" bereits korrekt abgefangen werden, und im weiteren Verlauf dann wie "jpg" behandelt werden.

Reply:
@aeberhard da fällt mir gerade auf, das strtolower aus der Zeile hier drunter, sollte eigentlich besser schon direkt in die erste `$ext`-zeile, oder?

Reply:
Ah OK, hatte versehentlich eine alte Version des Effekts angeschaut.